examples/image-component/pages/layout-fill.js

```diff
@@ -1,30 +1,26 @@
 import Image from 'next/image'
 import ViewSource from '../components/view-source'
+import mountains from '../public/mountains.jpg'
```

Comment:
Do we want to us lowercase or should we use uppercase to match other imports?

Reply:
Lowercase is correct, uppercase is for React components